kubernetes/configure

```diff
@@ -35,3 +42,48 @@ if [[ `juju status keystone --format=json 2>/dev/null| jq '.machines| length'` -
 fi
 
 kubectl --namespace kube-system get pods
+
+if [[ $profile == "stsstack" ]]; then
+    exit 0
+fi
+
+# Configure proxy and docker for all worker nodes
+for worker in $(juju status --format json | jq -r '.applications."kubernetes-worker".units | keys[]' | paste -sd " "); do
+    echo "Waiting worker $worker to be ready"
+    while [ "$(juju ssh $worker 'dpkg -l|grep containerd' &> /dev/null; echo $?)"  -ne 0 ]; do
+        sleep 5;
+        echo "Waiting for worker $worker to be online";
+    done
+    echo "done"
+done
+
+cat << EOF > $HOME/containerd-proxy.conf
+[Service]
+Environment="HTTP_PROXY=http://squid.internal:3128"
+Environment="HTTPS_PROXY=http://squid.internal:3128"
+Environment="NO_PROXY=127.0.0.1,10.0.0.0/8,192.168.0.0/16,172.16.0.0/12,registry.jujucharms.com,rocks.canonical.com,autocert.canonical.com,keystone.ps6.canonical.com,neutron.ps6.canonical.com,nova.ps6.canonical.com,glance.ps6.canonical.com,api.charmhub.io"
+EOF
+
```

Comment:
I would add an echo here telling users why/how to register.
Something along the lines:

`We need to add the docker credentials to your cloud in order not to be rate limited. You can create your account on https://hub.docker.com/".`